### PR TITLE
refactor: moved token detail price into chart

### PR DESCRIPTION
**Pull request by cartcrom**
Moved the token price & delta percentage from the tokendetails component into the price chart component & overlayed them onto the chart svg so that crosshair will appear when hovering over the whitespace above the chart

### CODE REVIEW

File: src/components/Charts/PriceChart.tsx [hunk 1]
```diff
@@ -30,13 +28,35 @@ function getDelta(start: number, current: number) {
...
+  font-size: 36px;
+  line-height: 44px;
+`
+export const DeltaContainer = styled.div`
+  height: 16px;
+  display: flex;
+  align-items: center;
+`
+const ArrowCell = styled.div`
+  padding-left: 2px;
```
Comment (vm) [inline src/components/Charts/PriceChart.tsx:49]: technically we don't allow 2px, lmk if we can avoid

File: src/components/Charts/PriceChart.tsx [hunk 2]
```diff
@@ -30,13 +28,35 @@ function getDelta(start: number, current: number) {
...
+`
+
 interface PriceChartProps {
   width: number
   height: number
 }
 
 export function PriceChart({ width, height }: PriceChartProps) {
-  const margin = { top: 40, bottom: 50 }
+  const margin = { top: 80, bottom: 20, crosshair: 70 }
```
Comment (vm) [inline src/components/Charts/PriceChart.tsx:59]: can you do 72 haha

File: src/components/Charts/PriceChart.tsx (hunk 1 above)
Comment (cartcrom) [reply src/components/Charts/PriceChart.tsx:49]: I copied this over from TokenDetail

Comment (vm) [reply src/components/Charts/PriceChart.tsx:49]: ah ok

File: src/components/Charts/PriceChart.tsx (hunk 2 above)
Comment (vm) [reply src/components/Charts/PriceChart.tsx:59]: if copied all good

**Review by vm — APPROVED** [review]

**Pull request merged**